core/contracts/common/ERC20Mintable.sol

```diff
@@ -0,0 +1,93 @@
+pragma solidity ^0.5.0;
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import "./AccessControl.sol";
+
+/**
+ * @dev Extension of {ERC20} that adds a set of accounts
+ * which have permission to mint (create) new tokens as they see fit.
+ *
+ * At construction, the deployer of the contract is the only admin capable of adding new minters and is initially the only minter.
+ */
+contract ERC20Mintable is ERC20, AccessControl {
```

Comment:
IMO, we should only implement the methods that we're going to use. See https://github.com/UMAprotocol/protocol/blob/904e0454b1bc09903add91fe0b562f32c7de0faf/core/contracts/oracle/implementation/VotingToken.sol#L27-L47 for a simple implementation of the mint and burn methods.

Reply:
Regarding which methods I want to include, I plan to expose:
- `mint`
- `isMinter`
- `setMinter`

The latter two are not used in any contracts but they are very useful for tests